problems/1.types+operators.js

```diff
@@ -2,19 +2,21 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  let value = number 
+  return (value === number )? true : false
+    
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
+  function isTruthy(val){
+    return (4 === 4) || (2 === 2)? true : false
```

Comment:
This will always return true. YOu're checking if 4 is equal to 4. Of course it is. Instead we need to be checking val is truthy or falsey. Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false.